acid/features/history/tests/test_buildsets.py

```diff
@@ -1,25 +1,25 @@
 # -*- coding: utf-8 -*-
-import unittest
+from unittest import TestCase
 from unittest.mock import MagicMock, patch
 
-from dashboard.exceptions import PageOutOfRange
-from dashboard.history import BuildSetsPaginated
+from ..exceptions import PageOutOfRange
+from ..service import BuildSetsPaginated
 
 
-class TestBuildSets(unittest.TestCase):
-    @patch('dashboard.model.ZuulBuildSet.get_for_pipeline')
+class TestBuildSets(TestCase):
+    @patch('acid.features.history.model.ZuulBuildSet.get_for_pipeline')
```

Comment:
You should use another technic for mocking ZuulBuildSet class. We want to get rid of `acid.features` prefix from all features patches. It can be done this way:
1. Import relatively Class you want to patch
3. Use `@patch.object` method
```python
from ..model import ZuulBuildSet

class TestBuildSets(unittest.TestCase):
    @patch.object(ZuulBuildSet, 'get_for_pipeline')
    def test_create_buildsets_history_object(self, get_for_pipeline):
        buildsets = BuildSetsPaginated(pipeline="foo", per_page=20)
        self.assertEqual(buildsets.per_page, 20)
```
It works as expected, I checked.

I will not comment out every line with the same snippet. 
Update all decorators in features: `auth/tests`, `status/tests`, `history/tests`.